src/main/java/com/flightstats/hub/spoke/SpokeContentDao.java

```diff
@@ -58,4 +74,28 @@
             throw e;
         }
     }
+
+    Optional<ChannelContentKey> getOldestItem(SpokeStore store) {
+        String storePath = HubProperties.getSpokePath(store);
+        logger.trace("getting oldest item from " + storePath);
+        // expected result format: YYYY-MM-DD+HH:MM:SS.SSSSSSSSSS /mnt/spoke/store/channel/yyyy/mm/dd/hh/mm/ssSSShash
+        String command = String.format(GET_OLDEST_ITEM_COMMAND, storePath);
+        String result = StringUtils.chomp(commander.runInBash(command, 3));
```

Comment:
Good call. I've been spoiled by the newer IntelliJ that provides context in that scenario.

<img width="878" alt="screen shot 2018-09-25 at 11 02 38 am" src="https://user-images.githubusercontent.com/161094/46033276-aaf9a800-c0b2-11e8-8ccd-a11090f28662.png">

Reply:
I think I actually turned that off since it kept telling me stupidly obvious things :). But that is a good point, if the IDE is smart enough to tell you the necessary context, the magic number isn't so magic.

Still, constants make the code reviews easier and gives a spot for a comment like ```// If we can't get an answer back in 3 seconds, something must be seriously hosed.```